doc/src/environment-variables.md

```diff
@@ -29,6 +29,12 @@
   determines the directory that traces will be written too. Traces are of the
   form PID.trace. Traces can be read by the Catapult project [tracing viewer].
 
+- `RUSTUP_TERM_COLOR` (default: `auto` in TTY, `never` otherwise) Controls whether colored output is used in the terminal. Possible values:
```

Comment:
I don't know how to parse the content in the brackets. Are you describing how `auto` works? If so perhaps just state the default, and describe the `auto` behaviour below.

Reply:
Done.